onnxruntime/core/providers/cpu/quantization/quantize_linear.cc

```diff
@@ -264,16 +293,28 @@ Status DequantizeLinear<T>::Compute(OpKernelContext* ctx) const {
 
 #define REGISTER_QUANTIZELINEAR(T)                                          \
   ONNX_CPU_OPERATOR_TYPED_KERNEL(                                           \
+      QuantizeLinear,                                                       \
+      21,                                                                   \
+      T,                                                                    \
+      KernelDefBuilder()                                                    \
+          .TypeConstraint("T1", {DataTypeImpl::GetTensorType<float>(),      \
+                                 DataTypeImpl::GetTensorType<MLFloat16>()}) \
+          .TypeConstraint("T2", DataTypeImpl::GetTensorType<T>()),          \
+      QuantizeLinear<T>);
+
+#define REGISTER_QUANTIZELINEAR_VERSIONED(T)                                \
```

Comment:
(As above): may be using `VERSION_20` is better than `VERSIONED`?

Reply:
We discussed and agreed to handle the rename separately. I'll do it the follow-up int4 PR.